jira_agile_metrics/config_test.py

```diff
@@ -523,7 +516,6 @@ def test_config_to_options_extends_blocked_if_no_explicit_working_directory():
 
     Backlog column: Backlog
     Committed column: Committed
-    Final column: Test
```

Comment:
Again - changes a test that would have passed previously.

Reply:
It would be safe to leave that there because it's not used and would be ignored.